chapters/numbers-to-indices/discrete-random-vars-binomial-distribution/reading/README.md

```diff
@@ -1 +1,548 @@
 # Discrete Random Variables and the Binomial Distribution
+
+## Simple Probabilities
+
+### Details
+
+Of all the possible 3-digit strings, $\binom{3}{x}$ of them have $x$ heads.
+So the probability of landing $x$ heads is $\binom{3}{x}p^x(1-p)^{3-x}$.
+
+### Examples
+
+:::info Example
+
+Consider a biased coin which has probability $p$ of landing heads up.
+If we toss this coin three independent times the possible outcomes are:
+
+$$\begin{array}{c c c} \hline \text{sequence} & \text{probability} & \text{Number of heads}\\ \hline \text{HHH} & p \cdot p \cdot p=p^3 & 3\\ \text{HHT} & p^2(1-p) & 2\\ \text{HTH} & p^2(1-p) & 2 \\ \text{HTT} & p(1-p)^2 & 1\\ \text{THH} & p^2(1-p) & 2\\ \text{THT} & p(1-p)^2 & 1\\ \text{TTH} & p(1-p)^2 & 1\\ \text{TTT} & (1-p)^3 & 0\\ \hline \end{array}$$
+
+:::
+
+:::info Example
+
+It is also possible to aggregate these values into a table and describe only the number of heads obtained:
+
+$$\begin{array}{c c} \hline \text{Heads} & \text{Probability}\ p(x)\\ \hline 0 & (1-p)^3\\ 1 & 3p(1-p)^2\\ 2 & 3p^2(1-p)\\ 3 & p^3 \\ \hline \end{array}.$$
+
+If we are only interested in the number of heads, then this table describes a **probability mass function** $p$, namely the probability $p(x)$ of every possible outcome $x$ of the experiment.
+
+:::
+
+:::info Example
+
+Given that a year is 365 days and each day has the same probability of being someone's birthday.
+What's the probability of at least 2 people sharing a birthday in a group of 25 people?
+Now, calculating each of the possible outcomes could become very tedious.
+That is calculating the odds that 2 people share a birthday, 3 people, 4 people, etc.
+So instead we try to find out the odds that no one in the group shares a birthday and subtract those odds from 1 (100%).
+
+First, let's look at the odds of only two people having distinct birthdays.
+
+$$\frac{365}{365}\cdot\frac{364}{365} = 0.9973$$
+
+Person one can be born on any day and the odds of having a distinct birthday are therefore 1.
+The next person can be born on everyday but the 1 the other person was born on, so 364 days
+
+Now let's say we add the 3rd person and calculate his/her odds of having a distinct birthday.
+
+$$\frac{365}{365}\cdot\frac{364}{365}\cdot\frac{363}{365} = 0.9918$$
+
+This can also be rewritten as
+
+$$\frac{365\cdot364\cdot363}{365^3}$$
+
+And we can do this on and on for all the `25` people we are interested in.
+But that may also become a bit tedious.
+So we use factorials instead.
+So instead of doing
+
+$$\frac{365\cdot364\cdot363\dots \cdot341}{365^{25}}$$
+
+we do
+
+$$\frac{\frac{365!}{340!}}{365^{25}}=0.4313$$
```

Comment:
```suggestion
$$\displaystyle\frac{\frac{365!}{340!}}{365^{25}}=0.4313$$
```
There seems to be an issue when stacking many "levels" (such as these fractions) of the same equation. The font becomes very small making the equation difficult to read. `\displaystyle` makes the font readable.